fbzmq/async/ZmqEventLoop.cpp

```diff
@@ -37,27 +42,35 @@ ZmqEventLoop::ZmqEventLoop(
   if ((callbackFd_ = eventfd(0 /* init-value */, EFD_NONBLOCK)) < 0) {
     LOG(FATAL) << "ZmqEventLoop: Failed to create an eventfd.";
   }
-
+  int callbackFD = callbackFd_;
+  int signalFd = signalFd_;
+#else
+  createPipe(signalFds_);
+  createPipe(callbackFds_);
```

Comment:
we can use kqueue in here if you guys prefer, but I think it's a overkill.

Reply:
TBH don't know about kqueue. Glad to learn with this change, it seems superior than epoll. Lol, side effect of being confined to linux.